### ISSUE TITLE
Wikia Thinks They Paid for My Computer Since Today

### ISSUE THREAD

**Issue by Zetawilk** [reek/anti-adblock-killer#2981]
http://jojo.wikia.com/wiki/Main_Page
http://imgur.com/a/sHuVE
Mozilla Firefox
Greasemonkey
Adblock Plus
Adblock Warning Removal List, EasyList, AakList (Anti-Adblock Killer)

**Comment by uBlock-user** [comment]
Force update your filters

**Comment by Zetawilk** [comment]
My Adblock Plus filters have been updated today. The erroneous behavior on the above website still occurs.

**Comment by uBlock-user** [comment]
Nevermind, it's patched on uBlock Origin, that's why I wasn't able to reproduce. For ABP - https://forums.lanik.us/viewtopic.php?f=62&t=35491&p=110832&hilit=wikia.com

**Comment by Zetawilk** [comment]
I hadn't heard of uBlock Origin until now. I'll be sure to check it out, though I've read implications that having multiple adblockers active at the same time can cause conflicts. 

That link didn't have very much in the way of overt directions, so I poked around ABP (AdBlock Plus). From `Filter Preferences > Custom Filters > Add Filter Group > Actions > Show/hide Filters` I added the three code lines in Intense's post (110786) individually using "Add Filter". I then visited the problem website and so far its content has not been censored by corporate imperialism. 

Thank you for your assistance. Hopefully ABP will notice this issue's essential fix has not been incorporated into their extension yet and will remedy the problem rather than mistake it for being fixed yet just because the solution is out there.

**Comment by uBlock-user** [comment]
> I've read implications that having multiple adblockers active at the same time can cause conflicts.

Yes, they do. Either run uBO or ABP, not both.

> ABP will notice this issue's essential fix has not been incorporated into their extension yet and will remedy the problem rather than mistake it for being fixed yet just because the solution is out there.

ABP is the extension. Easylist maintains filterlist which ABP uses, that means ABP has got nothing to do with it. Report on Easylist forums which is here - https://forums.lanik.us/viewforum.php?f=62, if you're using ABP or if you switch to uBO, then here - https://github.com/uBlockOrigin/uAssets/issues and I might also be able to help you further if you decide to use uBO, though I will leave that upto you. 

Sidenote - Great title, I like your sense of humour 👍

**Comment by Zetawilk** [comment]
> ABP is the extension. Easylist maintains filterlist which ABP uses, that means ABP has got nothing to do with it. Report on Easylist forums which is here - https://forums.lanik.us/viewforum.php?f=62, if you're using ABP or if you switch to uBO, then here - https://github.com/uBlockOrigin/uAssets/issues and I might also be able to help you further if you decide to use uBO, though I will leave that upto you.

You're right--thank you for clarifying that for me, and thanks again so much for the sources you've provided. 

I'm a little uncertain about closing this in case someone else pops in that needs clarification, or to augment something that doesn't work for everyone.

**Comment by uBlock-user** [comment]
Try 

```
@@||wikia.com^$generichide
/abtesting/
```
